src/config/dark-sites.config

```diff
@@ -468,6 +480,7 @@ viduro.xyz
 villains.fandom.com
 vimm.net
 vinesauce.com
+vipline.co.uk
```

Comment:
Vind the yellow pretty bright unsure about this one..

Reply:
Dark reader certainly makes a huge difference, will remove it.